monkey/monkey_island/cc/setup/agent_event_handlers.py

```diff
@@ -22,3 +29,16 @@ def _subscribe_and_store_to_event_repository(container: DIContainer):
         container.resolve(ICredentialsRepository)
     )
     agent_event_queue.subscribe_type(CredentialsStolenEvent, save_stolen_credentials_subscriber)
+
+
+def _subscribe_ping_scan_event(container: DIContainer):
+    # Mypy don't handle cases where abstract class is passed as Type[...]
+    # https://github.com/python/mypy/issues/4717
```

Comment:
This will be happening all over the place. I'm not sure if this comment adds anything or how do we explain the type ignores

Reply:
1. Can we put this ignore on the function instead of the callers? This would resolve all of the duplication (if it works).
2. We should include the error code in the ignore statements.